native/core/src/events/settings/user.rs

```diff
@@ -16,6 +16,7 @@ pub enum AutoDownload {
 pub struct AppChatSettings {
     #[serde(default, skip_serializing_if = "Option::is_none")]
     pub auto_download: Option<AutoDownload>,
+    pub typing_notice: Option<bool>,
```

Comment:
```suggestion
    #[serde(default, skip_serializing_if = "Option::is_none")]
    pub typing_notice: Option<bool>,
```

or old settings will not be deserialized properly